src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -69,14 +69,14 @@ public void testSaveAndLoadUsers() throws PersistentDataStoreException {
 
   @Test
   public void testSaveAndLoadConversations() throws PersistentDataStoreException {
-    UUID idOne = UUID.randomUUID();
-    UUID ownerOne = UUID.randomUUID();
+    UUID idOne = UUID.fromString("10000000-2222-3333-4444-555555555555");
+    UUID ownerOne = UUID.fromString("10000001-2222-3333-4444-555555555555");
     String titleOne = "Test_Title";
     Instant creationOne = Instant.ofEpochMilli(1000);
     Conversation inputConversationOne = new Conversation(idOne, ownerOne, titleOne, creationOne);
 
-    UUID idTwo = UUID.randomUUID();
-    UUID ownerTwo = UUID.randomUUID();
+    UUID idTwo = UUID.fromString("10000002-2222-3333-4444-555555555555");
+    UUID ownerTwo = UUID.fromString("10000002-2222-3333-4444-555555555555");
```

Comment:
this is the only owner UID identical to its matching "id" UID. Is this intentional?

Reply:
Good catch, this was a typo. Fixed.